README.md

```diff
@@ -10,27 +14,171 @@ Extracts browser and operating system information from the user agent string.
 
 Download dist files from repo directly or install it via npm. 
 
-### For development (Uncompressed)
+### Installation with npm
 
-You can download the uncompressed files for development
+The following command shows how to install egjs-agent using npm.
 
-- Latest : https://naver.github.io/egjs-agent/release/latest/dist/agent.js
-- Specific version : https://naver.github.io/egjs-agent/release/[VERSION]/dist/agent.js
+```bash
+$ npm install @egjs/agent
+```
 
-### For production (Compressed)
+### For development
 
-You can download the compressed files for production
+You can download the files for development
 
-- Latest : https://naver.github.io/egjs-agent/release/latest/dist/agent.min.js
-- Specific version : https://naver.github.io/egjs-agent/release/[VERSION]/dist/agent.min.js
+* Latest : https://naver.github.io/egjs-agent/release/latest/dist/agent.js
+* Latest(min): https://naver.github.io/egjs-agent/release/latest/dist/agent.min.js
+* Specific version : https://naver.github.io/egjs-agent/release/[VERSION]/dist/agent.js
+* Specific version(min):  https://naver.github.io/egjs-agent/release/[VERSION]/dist/agent.min.js
 
 
-### Installation with npm
+## Prepare for Client Hints!
 
-The following command shows how to install egjs-agent using npm.
+Chrome is currently planning to freeze user agents to improve user privacy.
```

Comment:
The term "currently" may be inappropriate because this README file will continue to be used in the future.
I think it might be more informative if you use a specific version instead.
> Chrome freezes user agents after version XX ...